.github/workflows/test-runner-resiliency-test.yml

```diff
@@ -0,0 +1,85 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Test Runner Resiliency Test
+
+on:
+  push
```

Comment:
if you commit this as-is, it will run a resiliency test against the ichang k8s namespace on *every* push. If multiple people push to their individual PRs simultaneously, it will cause concurrent resiliency tests against the same namespace. This feels like it will cause big problems.

Reply:
Updated the test trigger to match integration test workflow. Also changed namespace to `terra-wsmtest` as well.

Reply:
But, the pull_request trigger you have in place will run a resiliency test every time a pull request is opened, and every time a commit is pushed to an open pull request. This still feels like too much, and with a significant risk of multiple simultaneous resiliency tests colliding.

Reply:
Adjusted the trigger to run resiliency test once a day after midnight.